backend/scrapers/em2.js

```diff
@@ -0,0 +1,371 @@
+  );
+
+  if (cookiesNodig) {
+    await page.evaluate(() => {
+      document.querySelector("[name*='marketing']").click();
+      document.querySelector('.cookie__settings .cookie__process').click();
+    });
+    await this.waitTime(50);
+  }
+
+  await this.autoScroll(page);
+  await this.autoScroll(page);
+
+  let rawEvents = await page.evaluate(
+    // eslint-disable-next-line no-shadow
+    ({ workerData, unavailabiltyTerms }) =>
+      Array.from(document.querySelectorAll('.agenda-container .agenda-item'))
+        .filter((eventEl) => !eventEl.classList.contains('is-hidden'))
+        .map((eventEl) => {
+          const title = eventEl.querySelector('h3')?.textContent ?? '';
+          const res = {
+            anker: `<a class='page-info' href="${document.location.href}">${workerData.family} - main - ${title}</a>`,
+            errors: [],
+            title,
+          };
+
+          try {
+            const DDslashMM = eventEl.querySelector('.agenda-item-date')?.textContent ?? null;
+            const DD = DDslashMM.split("/")[0].trim();
+            const MM = DDslashMM.split("/")[1].trim();
+            res.startDate = `2024-${MM}-${DD}`;
+          } catch (caughtError) {
+            res.errors.push({
+              error: caughtError,
+              remarks: `date time faal ${title}.`,
+            });
+          }
+          res.shortText = eventEl.querySelector('.agenda-item-content p')?.textContent ?? '';
+          res.venueEventUrl = eventEl.querySelector('.more-button')?.href ?? null;
+
+          // TODO: Hoe te bepalen of show is uitverkocht?
+          const uaRex = new RegExp(unavailabiltyTerms.join('|'), 'gi');
+          res.unavailable = !!eventEl.textContent.match(uaRex);
+          res.soldOut =
+            !!eventEl
+              .querySelector('.program__status')
+              ?.textContent.match(/uitverkocht|sold\s?out/i) ?? false;
+
+          res.longText = eventEl.querySelector('.agenda-item-content p')?.textContent ?? null; // tijdelijk om in te controleren
+          return res;
+        }),
+    {
+      workerData,
+      unavailabiltyTerms: terms.unavailability,
+    },
+  );
+
+  rawEvents = rawEvents.map(this.isMusicEventCorruptedMapper);
+
+  const eventGen = this.eventGenerator(rawEvents);
+  // eslint-disable-next-line no-unused-vars
+  const checkedEvents = await this.rawEventsAsyncCheck({
+    eventGen,
+    checkedEvents: [],
+  });  
+
+  this.saveBaseEventlist(workerData.family, checkedEvents);
+    
+  const thisWorkersEvents = checkedEvents.filter(
+    (eventEl, index) => index % workerData.workerCount === workerData.index,
+  );
+    
+  return this.mainPageEnd({ stopFunctie, rawEvents: thisWorkersEvents });
+};
+// #endregion                          MAIN PAGE
+
+// #region [rgba(120, 0, 0, 0.1)]     SINGLE PAGE
+scraper.singlePage = async function ({ page, event }) {
+
+  const { stopFunctie } = await this.singlePageStart();
+
```

Comment:
Ik zie helemaal geen cookie bar bij em2? 
Deze code gebruik ik bij scrapers die vastlopen op een cookiebar. Zoals paradiso. 
Als er geen cookiebar is die alles of een gedeelte tegenhoud (youtubes etc) dan is het niet nodig te kijken naar cookies.